webclient/komponenta/models.py

```diff
@@ -101,7 +101,12 @@ def get_absolute_url(self):
                 args=[self.ident_cely[:-5], self.komponenta_vazby.dokumentacni_jednotka.ident_cely, self.ident_cely],
             )
         else:
-            return self.komponenta_vazby.casti_dokumentu.dokument.get_absolute_url()
+            if "3D" in self.ident_cely:
+                return self.komponenta_vazby.casti_dokumentu.dokument.get_absolute_url()
+            return reverse(
+                "dokument:detail-komponenta",
+                args=[self.komponenta_vazby.casti_dokumentu.dokument.ident_cely, self.ident_cely],
```

Comment:
Mělo by jít na ekvivalent: https://amcr-test.aiscr.cz/dokument/detail/C-TX-192600929/komponenta/C-TX-192600929-K001
Toto myslím vede na ekvivalent: https://amcr-test.aiscr.cz/dokument/detail/C-TX-192600929/cast/C-TX-192600929-D001

Reply:
tohle je za mne spravne, ak 3D, tak dokument detail, jinak dokument komponenta detail.

Reply:
Pardon, špatně jsem to četl.